cms_status_message/models.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# © 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
+from openerp.http import request
+from openerp import _
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    @property
```

Comment:
In my experience inheritance works... about translations: I'll add Italian translation and test it. Sounds good? :)